app/views/layouts/_user_menu.html.haml

```diff
@@ -21,7 +21,10 @@
     = link_to(destroy_user_session_path, :method=>'delete') do
       %span.fa.fa-minus
       Sign out
-- if can? :manage, Conference
+- if current_user.is_admin || (current_user.has_any_role? :admin, { :name => :organizer, :resource => :any },
```

Comment:
that's because only organizer can manage the conference model. cfp can manage at most the event model. still, he should be able to see admin/conference page.